src/Makefile

```diff
@@ -31,7 +32,7 @@ SED_EXTRA=-e 's/-Wall/-Wall -Wno-deprecated/'
 
 UNAME_S=$(shell uname -s)
 ifeq ($(UNAME_S),Darwin)
-  CPPFLAGS+= -I/usr/local/include -I/usr/local/opt/openssl/include
+  CPPFLAGS+= -UNOOPT -DNOOPT='[[clang::optnone]]' -I/usr/local/include -I/usr/local/opt/openssl/include
```

Comment:
Add a comment explaining why we need these options.

Reply:
No longer relevant, so removed.